docs/Using-SRIOV-and-Clear-Containers.md

```diff
@@ -0,0 +1,265 @@
+# Setup to use SR-IOV with Clear Containers
```

Comment:
I'd rather keep this as an action item that the user needs to perform.

Reply:
With work I've done on the Yocto Project, we use gerund form for headings so that came through here :) Thank you for the heads up that you not use gerund forms of verbs here - I'll make note of that for the future. Given the original heading "Setup to use SR-IOV..." I would suggest changing to "Set up to use SR-IOV..." because it is a set of tasks. I defer to @rcaballeromx